README.md

```diff
@@ -1,2 +1,91 @@
 # npmx-zephyr
-npmx integration with Zephyr
+
+<p align="center">
+    <img src="doc/_static/images/npm_family_logo_background.png" alt="NPMX LOGO" width="300" height="300">
+</p>
+
+## Overview
+npmx-zephyr repository provides a convenient way to work with Nordic Semiconductor's Power Management ICs (PMICs), by integrating [npmx library][npmx_repo]  into a [Zephyr][zephyr_site] RTOS environment.
+
+For the user convenience, this repository also includes a variety of sample applications that demonstrate the usage examples of the [npmx drivers][npmx_drivers].
+
+## Supported devices:
+ -    nPM1300 Engineering B
+
+## How to use
+
+### Repository initialization
+The recommended way to obtain repositories is by using the `west` tool. Information about installation, commands, configuration and more can be found in the [west tool documentation][west_doc].
+```bash
+west init -m https://github.com/NordicSemiconductor/npmx-zephyr
+cd nmpx-zephyr
+west update
+```
+Alternatively, repositories can be obtained using plain `git`, but in this case special attention should be paid to revisions, both zephyr and npmx revisions should match with the ones specified in `west.yml`, moreover `npmx` should be placed in `zephyr/modules` subdirectory.
```

Comment:
Sorry, I'm not sure what change are you proposing - did you add "in this case" only? If so, please be aware that in RST, sentences written one under another will be displayed as one paragraph, hence end users will be aware that these three sentences describe one option. 
Otherwise, please simply insert your version and I will check it during the next review round :)